signalfx-org-metrics/metrics.yaml

```diff
@@ -873,10 +883,24 @@ sf.org.numDatapointsDroppedExceededQuotaByToken:
   metric_type: counter
   title: sf.org.numDatapointsDroppedExceededQuotaByToken
 
+sf.org.numDatapointsDroppedInvalid:
+  brief: Number of data points dropped because they didn't follow documented guidelines for data points.
+  description: |
+    Number of data points dropped because they didn't follow documented guidelines for data points. For example, the metric name was too long, the metric name included unsupported characters, or the data point contained no values.
+
+        * Dimension(s):  `orgId`
+        * Data resolution: 10 seconds
+  metric_type: counter
+  title: sf.org.numDatapointsDroppedInvalid
+
 sf.org.numDatapointsDroppedInTimeout:
-  brief: Infrastructure Monitoring internal metric
-  description: Reserved for Infrastructure Monitoring internal use only.
-  metric_type: null
+  brief: Number of data points Observability Cloud didn't attempt to create because your account was throttled or limited in the previous few seconds.
```

Comment:
Generally seems like we should use the term 'organization' instead of 'account' but I'm okay with whatever is normal in the docs.

Reply:
Thank you for this, Aaron! It seems that we have used "organization" in the past -- but there seems to also be a new push to use "account" because it is clearer for users. I'll work to reconcile so that we use one or the other, but not both.